client/components/tinymce/plugins/contact-form/plugin.js

```diff
@@ -3,18 +3,21 @@
  */
 import tinymce from 'tinymce/tinymce';
 import i18n from 'lib/mixins/i18n';
-import ReactDom from 'react-dom';
-import ReactDomServer from 'react-dom/server';
-import React from 'react';
-
+import React, { createElement } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { renderToStaticMarkup } from 'react-dom/server';
+import { Provider } from 'react-redux';
 /**
  * Internal Dependencies
  */
 import Gridicon from 'components/gridicon';
 import ContactFormDialog from './dialog';
+import { initContactForm, addDefaultField, removeField } from 'state/ui/editor/forms/actions';
```

Comment:
This line is causing the build to fail, as it should be `contact-form`, not `forms`.